serviceWorker.js

```diff
@@ -0,0 +1,214 @@
+//Cache polyfil to support cacheAPI in all browsers
+importScripts('./cache-polyfill.js');
```

Comment:
What is this and do I even need this? I don't think so.

Reply:
Yeah it's actually needed to make your PWA to work offline. I will add this file.

Reply:
But this is just a polyfill, isn't it? In modern browsers, it should work without it.

Reply:
I guess You are Correct! I just keep this, you know? for redundancy xD

Reply:
btw, https://prkhrv.github.io  check out this PWA version of Flappy bird I made :)

Reply:
Best approach! 😄 

We will just try it out and can remove that later.